docs/wiki/Simnet-Testing.md

```diff
@@ -291,3 +291,27 @@ During this process, `dexc` should log messages similar to the following:
 ```
 
 For rapid setup, there is a script at `[repo root]/client/cmd/dexcctl/simnet-setup.sh` that has a reasonable default. You may also use this script as a template.
+
+### Acquire Simnet Funds
+
+#### DCR
+
+Get an address for your Decred simnet account using **dcrctl**.
+
+`dcrctl --simnet --wallet getnewaddress {accountname}`
```

Comment:
We have Decred native option now, so I think we should recommend getting a deposit address through the wallets view for all assets.